storage/migrations/13_analysis_node_height.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+-- Tracks the current (consensus) height of the node. TODO: expand to paratime heights.
+CREATE TABLE chain.latest_node_heights
+(
+  layer TEXT NOT NULL PRIMARY KEY,
+  height UINT63 NOT NULL
+);
+
+INSERT INTO chain.latest_node_heights (layer, height) VALUES ('consensus', 0);
```

Comment:
This is a temporary kludge. The larger issue here is that we currently do not have a way to cleanly test interval-based analyzers in the e2e tests because they run indefinitely and the e2e_test waits for all analyzers to complete. Planning to work on this in a follow-up PR so that we can test more endpoints and also remove this line. 

Note: The `node_stats` analyzer is disabled in the e2e tests for now, but I'm leaving the changes to add the `cache_volatile_endpoints` flag because it will be used in the aforementioned changes to support all analyzers in the e2e regression tests.

Reply:
Filed a clickup issue for it: https://app.clickup.com/t/8692jqubb